automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala

```diff
@@ -103,20 +108,17 @@ class FreeTrialSpec extends FreeSpec with BeforeAndAfterEach with Matchers with
         val billingAccountUponEnrollment = Google.billing.getBillingProjectAccount(billingProject.get)(trialAuthToken)
         assert(billingAccountUponEnrollment.nonEmpty, s"The user's project is not associated with a billing account.")
 
-        api.trial.terminateUser(testUser.email)
+        api.trial.terminateUser(testUser.email)(adminAuthToken)
 
         val billingAccountUponTermination = Google.billing.getBillingProjectAccount(billingProject.get)(trialAuthToken)
         val errMsg = "The trial user's billing project should have been removed from the billing account."
         assert(billingAccountUponTermination.isEmpty, errMsg)
-
-        registerCleanUpForDeleteTrialState()
```

Comment:
Why has this been deleted from all of these tests? Also, this one needs to get moved to the top.

Reply:
oh this is not suppose be included. I will revert changes.